docusaurus/docs/quickstarts/network/hosted.md

```diff
@@ -110,17 +109,15 @@ After the files are generated, you can then install them for use by systemd by r
 sudo cp "${ZITI_HOME}/${ZITI_EDGE_CONTROLLER_RAWNAME}.service" /etc/systemd/system/ziti-controller.service
 sudo cp "${ZITI_HOME}/${ZITI_EDGE_ROUTER_RAWNAME}.service" /etc/systemd/system/ziti-router.service
 sudo systemctl daemon-reload
-sudo systemctl start ziti-controller
-sudo systemctl start ziti-router
+sudo systemctl enable --now ziti-controller
+sudo systemctl enable --now ziti-router
 ```
 
 Now, both the controller and the edge router will restart automatically!  After a few seconds you can then run these
 commands and verify systemd has started the processes and see the status:
 
 ```bash
-sudo systemctl -q status ziti-controller --lines=0 --no-pager
-sudo systemctl -q status ziti-router --lines=0 --no-pager
-
+$ sudo systemctl -q status ziti-controller --lines=0 --no-pager

```

Comment:
this makes it ___way___ harder to copy/paste. before it was 'bad enough' because the output is in the same code block but now it's vastly more difficult to copy/paste these commands.

Reply:
fixed with separate "run me" blocks and "example output" blocks